src/clients/c++/library/request.h.in

```diff
@@ -159,6 +159,11 @@ class ServerHealthContext {
   /// \param ready Returns the readiness state of the server.
   /// \return Error object indicating success or failure of the request.
   virtual Error GetReady(bool* ready) = 0;
+  
+  /// Contact the inference server and get readiness state of a specific model.
+  /// \param ready Returns the readiness state of the model in the server.
+  /// \return Error object indicating success or failure of the request.
+  virtual Error GetModelReady(bool* ready, std::string model_name) = 0;
```

Comment:
I'd rather not add this. The way to determine this is to get the status of a model and we already have an API for that. That likely means that there is no way to access the V2 "model ready" endpoint via the client and that is ok.

Reply:
Sure, will remove this